kind/spec.yaml

```diff
@@ -1,9 +1,13 @@
 image: sighup/kind
 tags:
-  KUBECTL:
-    - v1.12.0
   KUSTOMIZE:
-    - 1.0.10
+    - 3.2.2
```

Comment:
Why we are switching to kustomize v3?

Do our current manifests work with the new version?

Reply:
I did not try all of the manifest we've got. I tried some of them, and it worked.

If there is something that did not work, we should fix it. I mean, stay in lower versions is an error.

Reply:
I think that since in the manifest we're not using any `secretGenerator` they should be fine anyway.

But the truth is that we've never tried and a good way to do it is having it under CI.